AndroidNativeKotlinTemplate/app/build.gradle.kts

```diff
@@ -7,6 +7,9 @@ plugins {
 
 dependencies {
     implementation("com.salesforce.mobilesdk:MobileSync:12.1.0")
+    // Uncomment when enabling log in via Salesforce UI Bridge API generated QR codes
```

Comment:
We'll have to see how we want to document using this in the template, but I used the phrase `when enabling log in via Salesforce UI Bridge API generated QR codes` on every touch point.  On a parallel, we need to see how we want to document the companion APEX code that backs this up as well.

Reply:
We can probably start by having the apex code in the documentation.
To make it easier we could have the server code in a sfdx project that is part of the template (this one or a new one) so that a dev can simply sfdx push the code to their org. We did that before in the [HybridLwcTemplate](https://github.com/forcedotcom/SalesforceMobileSDK-Templates/tree/dev/HybridLwcTemplate/server).
If we do that, maybe we should have QR login in its own template.

Reply:
We have so many templates, I was considering avoiding having another where the codebase is actually so similar to this one.  They do become toilsome to maintain in great numbers.  That said, this is one of the more complicated customizations we have for the template's user to make - more so than, say, enabling push notifications.

Reply:
If we end up having a server folder with a sfdx project that contains an apex page showing a QR code, it will make sense having a new template.